xga/generate/sas/lightcurve.py

```diff
@@ -248,7 +248,7 @@ def _lc_cmds(sources: Union[BaseSource, BaseSample], outer_radius: Union[str, Qu
             evt_list = pack[-1]
             # Sets up the file names of the output files, adding a random number
             dest_dir = OUTPUT + "{t}/{o}/{i}_{n}_temp_{r}/".format(t='xmm', o=obs_id, i=inst, n=source_name,
-                                                                   r=randint(0, int(1e+8)))
```

Comment:
Thanks for changing all the eROSITA randints - I forgot to convert all of them for the newer Python version. Just an FYI ones like this were fine, because it is being converted to int already.

Reply:
Ah, I'd done a blanket replace of 1e+8s inside of randint, I'll check closer in the future